_posts/2025-07-22-openvox-releases.md

```diff
@@ -0,0 +1,104 @@
+---
+layout: post
+title: New OpenVoxProject 8 releases!
+date: 2025-07-21
+github_username: bastelfreak
+---
+
+## New OpenVoxProject 8 releases!
+
+After lots of development, we have a new round of releases available!
+
+Our last releases are:
+
+* openvox-agent [8.19.2](https://github.com/OpenVoxProject/openvox-agent/releases/tag/8.19.2)
+* openvox-server [8.8.1](https://github.com/OpenVoxProject/openvox-server/blob/main/CHANGELOG.md#881-2025-03-19)
+* openvoxdb [8.9.1](https://github.com/OpenVoxProject/openvoxdb/blob/main/CHANGELOG.md#891)
+
+Our new releases:
+
+* openvox-agent [9.21.1](https://github.com/OpenVoxProject/openvox-agent/releases/tag/8.21.1)
+* openvox-server [8.10.0](https://github.com/OpenVoxProject/openvox-server/releases/tag/8.10.0)
+* openvoxdb will soon follow
+
+
+## Where are the releases?
+
+Right now, those are *beta* releases.
+That means that we haven't added them to our official repositories yet.
+You can download the individual packages here:
+
+* [openvox-agent](https://artifacts.voxpupuli.org/openvox-agent/8.21.1/)
+* [openvox-server](https://artifacts.voxpupuli.org/openvox-server/8.10.0/)
+
+We intend to keep the packages for at least one week in this beta state.
+When no issues are reported, we will move them to the official repositories after 2025-08-09.
+
+## Where do I report bugs and feature requests?
+
+Please come to our `#voxpupuli` channel on IRC or Slack (see [voxpupuli.org/connect](https://voxpupuli.org/connect/)).
+
+## What changed?
+
+We migrated from a local release process (as in: someone glues it together on their laptop and uploads the packages) to GitHub pipelines.
+
+### Release process changes
+
+* All our repositories now run their tests for each pull request
+* Each release is automatically documented in a CHANGELOG.md
+* We create a GitHub Release, also with a changelog
+* For each Ruby gem release to rubygems.org, we use [trusted publishing](https://guides.rubygems.org/trusted-publishing/)
+* All GitHub workflows were updated to use minimal permissions
+
+### Deprecated and broken operating systems
+
+We won't provide packages for the following EoL operating systems anymore:
+
+* Ubuntu 18.04
+* Ubuntu 20.04
+* Debian 10
+
+Debian broke [broke their images](https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=1109866) for Debian 11, so we cannot run tests for it right now.
+Also I've no intention to provide further OpenVox 7 packages.
```

Comment:
Maybe
> Also, the new build process does not provide packages for OpenVox 7 - we encourage users to migrate to OpenVox 8.

Reply:
Technically we could provide Openvox 7 packages. I Just think it is eol and not worth the effort.